zng/docs/spec.md

```diff
@@ -386,6 +412,7 @@ is interpreted as follows:
 | `bstring`  | variable |  UTF-8 byte sequence with `\x` escapes           |
 | `enum `    | variable |  UTF-8 bytes of enum string                      |
 | `ip`       | 4 or 16  |  4 or 16 bytes of IP address                     |
+| `port`     | XXX      |  XXX - to be filled in during PR review          |
```

Comment:
I noticed there's an entry for the `port` type in the section 3.1.1 Typedefs table, and it seems to be in the code, but it was't mentioned in this table here. I'm hoping someone who knows the code could plug in the correct suggestion. @alfred-landrum sounded like he's game to dig this up.